Server/mods/deathmatch/logic/CAPI.cpp

```diff
@@ -0,0 +1,81 @@
+#include "StdInc.h"
+#include "CAPI.h"
+#include "CGame.h"
+#include "CMainConfig.h"
+#include <SharedUtil.String.h>
+
+namespace Endpoints
+{
+    inline const std::string API_ENDPOINT = "/api";
+    namespace Server
+    {
+        inline const std::string PLAYERS = "/players";
+        inline const std::string MAX_PLAYERS = PLAYERS + "/max";
```

Comment:
Given that these variables are within cpp file, what do you exactly expect from inline specifier here?

Reply:
I thought about moving them to header file and forgot to remove `inline`

Reply:
Have you considered const static `std::string_view`?

Reply:
Nope, will take that into consideration